sdk/maintenance/Azure.ResourceManager.Maintenance/src/Custom/MaintenanceConfigurationData.cs

```diff
@@ -1,8 +1,6 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-// <auto-generated/>
```

Comment:
I did not see anything changed in this file comparing with its generated version.
Do we have any?
If not, please remove this file and keep this file generated so that we could get new properties in future generation.

Reply:
The custom code is in the `MaintenanceConfigurationData.Serialization.cs` file, which need to use `CodeGenSuppressType` to suppress the whole `MaintenanceConfigurationData` model, so have to copy this file to the `Custom` folder too.

Reply:
Fixed, use `CodeGenSuppress` directly overwrite the serialization method.